rules/sinks/third_parties/sdk/google/java.yaml

```diff
@@ -20,14 +20,191 @@ sinks:
       - "(?i)(com[.]google[.]api[.]services[.]analytics|com[.]google[.]android[.]gms[.]analytics).*"
     tags:
 
-  - id: ThirdParties.SDK.Google.Cloud
-    name: Google Cloud
+  - id: ThirdParties.SDK.Google.Cloud.Storage
+    name: Google Cloud Storage
+    domains:
+      - "storage.cloud.google.com"
+    patterns:
+      - "(?i)(com[.]google[.]cloud[.]storage).*"
+    tags:
+
+  - id: ThirdParties.SDK.Google.Cloud.Datastore
+    name: Google Cloud Datastore
     domains:
       - "cloud.google.com"
     patterns:
-      - "(?i)(com[.]google[.]cloud[.]storage|com[.]google[.]cloud[.]dataflow|com[.]google[.]cloud[.]datastore|com[.]google[.]cloud).*"
+      - "(?i)(com[.]google[.]cloud[.]datastore|com[.]google[.]cloud[.]grpc).*"
     tags:
-    
+
+  - id: ThirdParties.SDK.Google.Cloud.Spanner
+    name: Google Cloud Spanner
+    domains:
+      - "cloud.google.com"
+    patterns:
+      - "(?i)(com[.]google[.]cloud[.]spanner|com[.]google[.]cloud[.]grpc).*"
```

Comment:
I see `grpc` rule again here, this will result in match across rule, Rule should be atomic and should result in single match

Reply:
removed it